src/template-parameters.json

```diff
@@ -264,32 +264,13 @@
         "name": "cookieDuration",
         "help": "The cookie duration in milliseconds (default to `15552000000`, 6 months).",
         "type": "TEXT",
-        "simpleValueType": true,
-        "valueValidators": [
-          {
-            "type": "POSITIVE_NUMBER"
-          }
-        ]
+        "simpleValueType": true
       },
       {
         "displayName": "User Token",
         "name": "initialUserToken",
         "help": "Set an initial user identifier applied to subsequent events.",
         "type": "TEXT",
-        "simpleValueType": true,
-        "valueValidators": [
-          {
-            "args": ["^.{1,64}$"],
-            "type": "REGEX"
-          }
-        ]
-      },
-      {
-        "displayName": "Search Insights Source URL",
-        "name": "searchInsightsSource",
-        "defaultValue": "https://cdn.jsdelivr.net/npm/search-insights@1.8.0",
-        "help": "The source URL of the Search Insights library.",
-        "type": "TEXT",
         "simpleValueType": true
       }
```

Comment:
This makes an `input` box on GTM. Its default value was that, but in order to update the version, our customers had to update the input value. It's not something we were able to control